apps/console/src/features/identity-providers/components/identity-provider-list.tsx

```diff
@@ -292,47 +292,61 @@ export const IdentityProviderList: FunctionComponent<IdentityProviderListPropsIn
                 dataIndex: "name",
                 id: "name",
                 key: "name",
-                render: (idp: IdentityProviderInterface): ReactNode => (
-                    <Header
-                        image
-                        as="h6"
-                        className="header-with-icon"
-                        data-testid={ `${ testId }-item-heading` }
-                    >
-                        {
-                            idp.image
-                                ? (
-                                    <AppAvatar
-                                        size="mini"
-                                        name={ idp.name }
-                                        image={ idp.image }
-                                        spaced="right"
-                                        data-testid={ `${ testId }-item-image` }
-                                    />
-                                )
-                                : (
-                                    <AppAvatar
-                                        image={ (
-                                            <AnimatedAvatar
-                                                name={ idp.name }
-                                                size="mini"
-                                                data-testid={ `${ testId }-item-image-inner` }
-                                            />
-                                        ) }
-                                        size="mini"
-                                        spaced="right"
-                                        data-testid={ `${ testId }-item-image` }
-                                    />
-                                )
-                        }
-                        <Header.Content>
-                            { idp.name }
-                            <Header.Subheader data-testid={ `${ testId }-item-sub-heading` }>
-                                { idp.description }
-                            </Header.Subheader>
-                        </Header.Content>
-                    </Header>
-                ),
+                render: (idp: IdentityProviderInterface): ReactNode => {
+                    const isOrgIdp = (idp.federatedAuthenticators.defaultAuthenticatorId ===
```

Comment:
Shall we type this const

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3176